metafield.go

```diff
@@ -39,17 +39,18 @@ type MetafieldServiceOp struct {
 
 // Metafield represents a Shopify metafield.
 type Metafield struct {
-	ID            int64       `json:"id,omitempty"`
-	Key           string      `json:"key,omitempty"`
-	Value         interface{} `json:"value,omitempty"`
-	ValueType     string      `json:"value_type,omitempty"`
-	Namespace     string      `json:"namespace,omitempty"`
-	Description   string      `json:"description,omitempty"`
-	OwnerId       int64       `json:"owner_id,omitempty"`
-	CreatedAt     *time.Time  `json:"created_at,omitempty"`
-	UpdatedAt     *time.Time  `json:"updated_at,omitempty"`
-	OwnerResource string      `json:"owner_resource,omitempty"`
-	AdminGraphqlAPIID string  `json:"admin_graphql_api_id,omitempty"`
+	ID                int64       `json:"id,omitempty"`
+	Key               string      `json:"key,omitempty"`
+	Value             interface{} `json:"value,omitempty"`
+	ValueType         string      `json:"value_type,omitempty"` // deprecated in version 2021-07
```

Comment:
I wonder if instead of adding comments to the lines, we just link to the Shopify documentation at the top of the struct. That might make things a bit less clear, but also saves us from being liable with future changes

Reply:
Given this simply calls out deprecation at specific versions for anyone who's using this code, I think it's safe enough since those will not change for these versions, but we cannot be sure for the future.

I can also add a comment at the top of the struct linking to the Shopify documentation if that's helpful.

Reply:
I did a search through a few files and couldn't find anywhere where we would add comments like this, which is why I'm inclined to not include them. If you can find some area where we do that already, I would be more inclined to accept this change

Reply:
In the sake of consistency I've removed those comments.